pkg/vault/contracts/VaultExplorer.sol

```diff
@@ -292,7 +292,7 @@ contract VaultExplorer is IVaultExplorer {
 
     /// @inheritdoc IVaultExplorer
     function collectAggregateFees(address pool) external {
-        return _vault.collectAggregateFees(pool);
```

Comment:
Do we still want to expose this on the Vault Explorer, as it's not strictly a "Vault" operation anymore? Then the VaultExplorer would be just read only.

Reply:
Ok; I guess it makes it easier to test fees.